### DISCUSSION TITLE
Changing legend name

### DISCUSSION THREAD

**Discussion by Darecki1** [grafana/grafana#39501] [General]
How to change that naming to be able to override color separatly?
![image](https://user-images.githubusercontent.com/90243069/134303082-c30b4046-2228-4aca-aefd-00c55cea84d5.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
